axonivy-engine/10.0/Dockerfile

```diff
@@ -27,7 +27,10 @@ RUN apt-get update && \
     ln -s ${IVY_HOME}/logs /var/log/axonivy-engine
 
 ADD --chown=ivy:ivy ./docker-entrypoint.sh ${IVY_HOME}/bin/docker-entrypoint.sh
-RUN chmod u+x ${IVY_HOME}/bin/docker-entrypoint.sh 
+RUN chmod u+x ${IVY_HOME}/bin/docker-entrypoint.sh && \
+\
+    chgrp -R 0 ${IVY_HOME} && \
```

Comment:
@ivy-cst What do you think? The user is still (1000=ivy) but the group is now (0=root). Basically we don't rely on that fact that the group must be ivy, it's enough that the user is ivy.  Maybe this is a good change and will cause less troubles also in other environments?

Reply:
After discussion with @ivy-cst 

In this layer we should only modify the docker-entrypoint.sh. We should already set the group 0 on line 19 and line 24.